### PR TITLE
4L Training

### PR DESCRIPTION
**Pull request by rhemon**
Training 4 Layer Neural Network

### CODE REVIEW

**Comment by rhemon** [comment]
4800 - 2400 - 2400 - 4
2000 iterations
.008 learning rate

```
Accuracy with training set, 72.671875%
Accuracy with dev set, 75.25%
Accuracy in recognising mango: 74.7777777778
Accuracy in recognising apple: 68.5555555556
Accuracy in recognising orange: 89.2222222222
Accuracy in recognising pear: 50.0
```

**Comment by rhemon** [comment]
4800 - 2400 - 2400 - 4
2000 iterations
.01 learning rate

```
Accuracy with training set, 72.046875%
Accuracy with dev set, 73.75%
Accuracy in recognising mango: 74.7777777778
Accuracy in recognising apple: 65.3333333333
Accuracy in recognising orange: 89.5555555556
Accuracy in recognising pear: 50.0
```

**Comment by rhemon** [comment]
4800 - 2400 - 2400 - 4
2000 iterations
.009 learning rate
```
Accuracy with training set, 71.734375%
Accuracy with dev set, 74.0%
Accuracy in recognising mango: 75.2222222222
Accuracy in recognising apple: 64.1111111111
Accuracy in recognising orange: 89.3333333333
Accuracy in recognising pear: 50.0
```

**Comment by rhemon** [comment]
12 - 24 - 48 - 4
3000 iterations
.02 learning rate
```
Accuracy with training set, 64.328125%
Accuracy with dev set, 68.75%
Accuracy in recognising mango: 50.0
Accuracy in recognising apple: 50.0
Accuracy in recognising orange: 100.0
Accuracy in recognising pear: 50.0
```
[CircleCI Log](https://circleci.com/gh/rhemon/food_rec/68)
